project/apps/bff/src/app/users.controller.ts

```diff
@@ -0,0 +1,62 @@
+import { Body, Controller, Post, Get, Req, UseFilters, HttpStatus, Param } from '@nestjs/common';
+import { HttpService } from '@nestjs/axios';
+import { ApplicationServiceURL } from './app.config';
+import { Request } from 'express';
+import { LoginUserDto } from './dto/login-user.dto';
+import { AxiosExceptionFilter } from './filters/axios-exception.filter';
+import { ApiResponse, ApiTags } from '@nestjs/swagger';
+import { UserRdo } from './rdo/user.rdo';
+import { UserRole } from '@project/shared/app-types';
+
+@ApiTags('users')
+@Controller('users')
+@UseFilters(AxiosExceptionFilter)
+export class UsersController {
+  constructor(
+    private readonly httpService: HttpService
+  ) {}
+
+  @Post('login')
+  public async login(@Body() loginUserDto: LoginUserDto) {
+    const { data } = await this.httpService.axiosRef.post(`${ApplicationServiceURL.Users}/login`, loginUserDto);
+    return data;
+  }
+
+  @Post('refresh')
+  public async refreshToken(@Req() req: Request) {
+    const { data } = await this.httpService.axiosRef.post(`${ApplicationServiceURL.Users}/refresh`, null, {
+      headers: {
+        'Authorization': req.headers['authorization']
+      }
+    });
+
+    return data;
+  }
+
+  @ApiResponse({
+    status: HttpStatus.OK,
+    description: 'User found.'
+  })
+  @Get(':id')
+  public async show(@Req() req: Request, @Param('id') id: string) {
+    
+    const user: UserRdo = await this.httpService.axiosRef.get(`${ApplicationServiceURL.Users}/${id}`, {
+      headers: {
```

Comment:
здесь в ответ на запрос не user приходит а объект ответа, в котором есть поле data и вот там как рз информация хранится 
```suggestion
    const { data }= await this.httpService.axiosRef.get(`${ApplicationServiceURL.Users}/${id}`, {
      headers: {
```